packages/nocodb/src/lib/version-upgrader/ncUpgradeErrors.ts

```diff
@@ -0,0 +1,19 @@
+export function throwTimeoutError(e, timeoutErrorInfo) {
+  if (e.code === 'ETIMEDOUT') {
```

Comment:
For different database client, the error code may be something else and it can be `EHOSTDOWN` as well(https://github.com/nocodb/nocodb/issues/5147)